client/blocks/login/login-form.jsx

```diff
@@ -373,7 +373,7 @@ export class LoginForm extends Component {
 							{ preventWidows(
 								this.props.translate(
 									// To make any changes to this copy please speak to the legal team
-									'By continuing with any of the options below, ' +
+									'By creating an account, ' +
```

Comment:
Hi! I've found a possible matching string that has already been translated **26** times:
`translate( 'By continuing with any of the options below, you agree to our {{tosLink}}Terms of Service{{/tosLink}}.' )` *ES Score: 6*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).